database/index.js

```diff
@@ -100,9 +100,61 @@ const randomCities = (country, cities, cityIndex) => {
   return cityCollection;
 };
 
+const getLocations = (projectId) => {
+  let query = 'SELECT locations FROM randomLocations WHERE id = ?';
+
+  return new Promise((resolve, reject) => {
+    connection.query(query, [projectId], (err, results) => {
+      err ? reject(err) : resolve(results);
+    })
+  });
+};
+
+const getCountry = (countryIds) => {
```

Comment:
If I'm understanding this correctly, it's getCountries (plural), right? Similarly with getCity